integrations/launchdarkly/.port/resources/blueprints.json

```diff
@@ -153,5 +153,32 @@
         "many": false
       }
     }
+  },
+  {
+    "identifier": "featureFlagInEnvironment",
```

Comment:
```suggestion
    "identifier": "launchDarklyFeatureFlagInEnvironment",
```

Reply:
String is longer than the maximum length of 30.